### ISSUE TITLE
Cocoa theme doesn't provide the standard header for .md files

### ISSUE THREAD

**Issue by asyncial** [nishanths/cocoa-hugo-theme#98]
I tried to create a new blog page, an it works, but there is noch header information, like date, draft and title.
Hugo spits put a warning too:

```WARNING: date and/or title missing from archetype file "/home/*/src/blog/themes/cocoa/archetypes/default.md". 
From Hugo 0.24 this must be provided in the archetype file itself, if needed. Example:
---
title: "{{ replace .TranslationBaseName "-" " " | title }}"
date: {{ .Date }}
draft: true
---
```
Is this wanted behaviour?`

**Comment by lpancescu** [comment]
This certainly isn't intended, but I don't see the warning you've quoted anywhere, not even when running `hugo -v` on my own site. I'm running Hugo 0.25, though. Do you see it with the example site from the theme?

**UPDATE** Never mind, I see it when running `hugo new blog/test.md` (for the future, please provide the exact commands you're trying to run, not just their output).

**Comment by lpancescu** [comment]
Thanks for reporting this, fixed by commit 685cb1f
